rockspec/apisix-dev-1.0-0.rockspec

```diff
@@ -32,6 +31,7 @@ dependencies = {
     "luafilesystem = 1.7.0-2",
     "lua-tinyyaml = 0.1",
     "iresty-nginx-lua-prometheus = 0.20190917",
+    "lua-resty-jsonschema = 0.1",
```

Comment:
remove `rapidjson` and add `lua-resty-jsonschema` in copyright file.

Reply:
add todo list